.gitignore

```diff
@@ -7,15 +7,13 @@ foreman-installer*
 puppet-*
 boxes.yaml
 .idea
-scripts
 repo
-Gemfile.lock
+vagrant/Gemfile.lock
```

Comment:
Maybe this should be moved to ```vagrant/.gitignore``` along with the other vagrant specific ignores? Or at least grouped in this file.

Reply:
+1 to `vagrant/.gitignore`